obs-studio-server/source/nodeobs_service.h

```diff
@@ -294,6 +294,9 @@ class OBS_service
 	static bool resetAudioContext(bool reload = false);
 	static int  resetVideoContext(bool reload = false);
 
+	// Prepare video context configuration
```

Comment:
This comment line does not provide additional information so can be deleted or improved.